### PR TITLE
fix(cli): Correctly set NoLSI field when askDynamoLSIConfig returns false

### PR DESCRIPTION
**Pull request by bvtujo**
This change solves an issue where if a customer selected "No" when asked "Would you like to add any alternate sort keys to this DynamoDB table?", the `noLsi` option was not properly set and an erroneous line was injected into the CF template. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: internal/pkg/cli/storage_init_test.go [hunk 1]
```diff
@@ -632,10 +664,18 @@ func TestStorageInitOpts_Ask(t *testing.T) {
...
+				tc.wantedVars.prompt = opts.prompt
+				require.Equal(t, *tc.wantedVars, opts.initStorageVars)
+			}
 		})
 	}
 }
 
+func TestStorageInitOpts_AskOutputs(t *testing.T) {
+
+}
```
Comment (iamhopaul123) [inline internal/pkg/cli/storage_init_test.go:677]: why do we have this empty test func?

Comment (bvtujo) [reply internal/pkg/cli/storage_init_test.go]: Sorry, fixed. Added while I was looking for the best way to validate this and forgot to remove.

File: internal/pkg/cli/storage_init.go [hunk 2]
```diff
@@ -399,13 +399,20 @@ func (o *initStorageOpts) askDynamoLSIConfig() error {
...
+		}
+		// This will execute last in the loop if moreLSI is set to false by any confirm prompts.
+		if !moreLSI {
+			if len(o.lsiSorts) == 0 {
+				o.noLsi = true
+			} else {
+				o.noLsi = false
+			}
 			break
 		}
```
Comment (efekarakus) [inline internal/pkg/cli/storage_init.go:414]: can we have:
```suggestion
		if !moreLSI {
			o.noLsi = len(o.lsiSorts) == 0
			return nil
		}
```

Comment (efekarakus) [reply internal/pkg/cli/storage_init.go:414]: also can we update `noLsi` to `noLSI`

https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Comment (bvtujo) [reply internal/pkg/cli/storage_init.go]: Done.

**Pull request merged**